docs/labwc-actions.5.scd

```diff
@@ -99,11 +99,16 @@ Actions are used in menus and keyboard/mouse bindings.
 	Resize and move active window according to the given region.
 	See labwc-config(5) for further information on how to define regions.
 
-*<action name="NextWindow" />*
-	Cycle focus to next window.
-
+*<action name="NextWindow" />*++
 *<action name="PreviousWindow" />*
-	Cycle focus to previous window.
+	Cycle focus to next/previous window respectively.++
+	Default keybind for NextWindow is Alt-Tab.
+
+	The shift key is used to reverse direction while cycling.
+
+	The arrow keys are used to move forwards/backwards while cycling.
+
+	*Note:* Shift key should not be used in keybind for these 2 actions.
```

Comment:
This is a no-go for me, because I use `S-A-Tab` for `PreviousWindow` and I think that's common (therefore arguably expected) behaviour.

Reply:
Fixed, borrowed a little logic from your WIP.